### PR TITLE
Suggest notifications after ticket closure.

### PR DESCRIPTION
**Pull request by ktlim**
Notifications to community.lsst.org for significant features/bug fixes; release notes for anything affecting users.

### CODE REVIEW

**Comment by timj** [comment]
This would be good. I do scan through community looking for announcement posts like that when doing the weekly highlights. Can we also recommend a specific tag be used on community?

**Comment by ktlim** [comment]
Added your suggested tag.

File: processes/workflow.rst [hunk 1]
```diff
@@ -413,6 +413,9 @@ We **always use non-fast forward merges** so that the merge point is marked in G
    git push
 
 Once the merge has been completed, the developer should mark the JIRA ticket as **Done**.
+If this merge adds a significant feature or fixes a significant bug, it should be announced in the "DM Notifications" category of community.lsst.org with tag `dm-dev`.
```
Comment (jonathansick) [inline processes/workflow.rst]: Perhaps add some inline links here:

```
`DM Notifications category <https://community.lsst.org/c/dm/dm-notifications>`_
```

and

```
`dm-dev <https://community.lsst.org/tags/dm-dev>`_
```

**Comment by jonathansick** [comment]
Looking at how this renders (http://developer.lsst.io/en/tickets-dm-5975/processes/workflow.html#merging) I realized that the flow goes from Git merge procedure -> JIRA done -> announcements -> more on the big green button. I think it would be warranted to move this paragraph (even if it's only a few sentences) to its own section just following this "Merging" section? That would instantly give this part of the workflow more visibility to someone skimming the page.

```
.. _workflow-announce:

Announce the change
-------------------
```

**Comment by ktlim** [comment]
Incorporated @jonathansick 's suggestions.

**Comment by timj** [comment]
Shall we merge this then?

**Comment by jonathansick** [comment]
@frossie was also listed as a reviewer. I finished my review. I think we can just merge it.

**Comment by ktlim** [comment]
I wanted @frossie  to look at it from a process perspective rather than strictly a documentation perspective.  I should probably have stated so explicitly.

**Pull request merged**